ballista/rust/core/proto/ballista.proto

```diff
@@ -249,9 +249,10 @@ message LogicalPlanNode {
     RepartitionNode repartition = 9;
     EmptyRelationNode empty_relation = 10;
     CreateExternalTableNode create_external_table = 11;
-    AnalyzeNode analyze = 14;
     ExplainNode explain = 12;
     WindowNode window = 13;
+    AnalyzeNode analyze = 14;
```

Comment:
@alamb i reordered the node down here so it's easier to do the counting. I am guessing you put it next to `ExplainNode` because that's how they should be logically grouped? If so, maybe we could keep it next to `ExplainNode` and reorder the subsequent fields.

Reply:
I don't have a strong preference. `rust` will tell us when the cases are missing so the order in which they are defined is not all that critical in my opinion